doc/v2/howto/rnn/hierarchical_layer_en.rst

```diff
@@ -1,4 +1,88 @@
 Layers supporting hierarchical sequence as input
 ================================================
 
-TBD
+..	contents::
+
+summary
+=======
+
+The sequence is common data structure in the proccess of NLP(nature language proccess). A single word can be seem as a non-sequence input or a zero-layer input. The sentence make up by words is a one-layer sequence and the senten is a two-layer squence. 
```

Comment:
The sequence is common  -> A sequence is a common

Reply:
nature language proccess -> **Natural Language Processing**

Reply:
This line does not make sense -> `A single word can be seem as a non-sequence input or a zero-layer input.`

Reply:
@shanyi15

Reply:
get, thanks!